src/lolex-src.js

```diff
@@ -689,7 +689,8 @@ exports.install = function install(config) {
     clock.methods = config.toFake || [];
 
     if (clock.methods.length === 0) {
-        clock.methods = keys(timers);
+        // do not fake nextTick by default - GitHub#126
+        clock.methods = keys(timers).filter(function (key) {return key !== "nextTick";});
```

Comment:
This means we stub all the other timers by default, but this is not reflected in the docs, which erronously says the default is an empty array. Care to fix the docs? It's the line on `config.toFake` in the README.